### DISCUSSION TITLE
Homepage

### DISCUSSION THREAD

**Discussion by pioneeriham** [minbrowser/min#2336] [Ideas]
Is it possible to open Min to a specific link/website on startup, every time? And maybe if new tabs could be set to open to this page too?

**Comment by VoxAndrews** [comment]
Bumping this as this would actually be a neat thing to have. For myself it's not that big of an issue, but I'm using Min as my browser-of-choice for a Ubuntu OS I'm customizing for some people and they've asked if they can have it start on Google as its their search engine of choice. They're a bit older, and just having that extra step taken for them, at least the option to set it up for them like that, helps people out a tonne who might not have as much experience with using the internet or computers.

**Comment by Tedd-Codes** [comment]
This could be simple to add, please make a feture request in the issues tab

**Comment by Chickaboo** [comment]
#2648
